### PR TITLE
Media query Screen 768px

### PR DESCRIPTION
**Pull request by UmerZia**
### In this branch I:

- Add mediaQuerys according to 768px wide screen.
- Made my webpage fully responsive.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,25 +7,36 @@
...
+        <h2 class="sub-heading">I am a Software Developer</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+
       <div>
         <ul class="icon">
```
Comment (Lucas-Masaba) [inline index.html:41]: - [ ] Please increase the spacing between the paragraph and the icons as seen in the Figma template 👍 
### Your Implementation
image
### Figma 
image

File: style.css [hunk 2]
```diff
@@ -417,10 +432,170 @@
...
+  .hand-burger {
+    visibility: hidden;
+    height: 0;
+    width: 0;
+  }
+
+  .main-card img {
+    width: 50%;
+    height: auto;
+  }
```
Comment (Lucas-Masaba) [inline style.css:509]: - [ ] Please make this image larger as seen in the Figma template.
### Your Design
image
### Figma template
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @UmerZia 👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Good-looking UI 🧹 
- Linter checks passing ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (UmerZia) [reply index.html:41]: I have applied the changes as per instructions.

File: style.css (hunk 2 above)
Comment (UmerZia) [reply style.css:509]: I have increased the width % of the image.

**Review by Kandy-Peter — APPROVED** [review]
Hi @UmerZia,

### APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

##  Optional suggestion

- [ ] OPTIONAL: It would be better if you enlarge your section's height. Their heights are too small. You can increase to make them take, each one, the all-screen size on non-scroll.

 
# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**